web-api/openapi/back_office.yaml

```diff
@@ -47,11 +47,18 @@ paths:
       tags:
         - back office
       summary: Add documents to the system
-      description: |-
-        Add documents to the system. The system will create a representation of the document
-        that will be used to match it against the preferences of a user.
-        Important note: currently we allow up to a maximum of 100 documents per batch size.
-        If you need to add more, then please split up the total amount of documents in separate calls, where each call contains at maximum 100 documents.
+      description: |
+        Add documents to the system.
+
+        The system will create a representation of the document that will be used to match it against the preferences of a user.
+
+        **Important note:** Currently we allow up to a maximum of 100 documents per batch size. If you need to add more, then
+        please split up the total amount of documents in separate calls, where each call contains at maximum 100 documents.
+
+        **Important note:** Documents which have no `publication_date` (in `properties`) or a `publication_date` in the future are
+        treated as unpublished and as such are filtered out from personalized documents. This allows ingesting documents which e.g.
```

Comment:
I don't think we want to filter out the documents that don't have a published date if `published_after` is not set.

Reply:
From slack:

-  with `published_after`  filter `after <= publication <= now`  (like currently, this excludes `publication == null`)
-  without filter `after == null || after <= now`